.github/workflows/ci.yml

```diff
@@ -34,17 +30,14 @@ jobs:
             - 'tests/**'
           embedchain:
             - 'embedchain/**'
-            - 'embedchain/tests/**'
-            - 'embedchain/examples/**'
 
   build_mem0:
     needs: check_changes
-    if: ${{ needs.check_changes.outputs.mem0_changed == 'true' || (needs.check_changes.outputs.mem0_changed == 'false' && needs.check_changes.outputs.embedchain_changed == 'false') }}
+    if: needs.check_changes.outputs.mem0_changed == 'true'
```

Comment:
What is the reasoning behind this?

I think what we want to do here is that:
1. Run mem0 tests if mem0 changes are there
2. Run embedchain tests if embedchain related changes are there

Reply:
Also please see the tests, for this PR only Mem0 tests ran

Reply:
Ah got it. Cool.